### DISCUSSION TITLE
unable to find input file

### DISCUSSION THREAD

**Discussion by sherrieF** [dtcenter/METplus#1237] [File I/O] [answered]
I am working on Hera.  I am trying to run the METplus gridstat.  The directory on Hera that I am trying to run from is /home/Sherrie.Fredrick/METplus/gridstat.  This is where my conf file is.  It is called 
GridStat.conf .  I also have a defaults.conf in this directory.   I am running the following from the
command line: /contrib/METplus/METplus-4.0.0/ush/run_metplus.py -c defaults.conf GridStat.conf

It seems that METplus is unable to find my input files.  Could not find FCST_INPUT file /scratch2/BMC/gsienkf/Sherrie.Fredrick/C96_replay_winter/2015120218/GFSPRS.GrbF012.grib2" using template {init?fmt=%Y%m%d%H}/GFSPRS.GrbF{lead?fmt=%3H}.grib2"

The input file is there.
-rw-r--r-- 1 Sherrie.Fredrick gsienkf 63255953 Oct 22 16:08 /scratch2/BMC/gsienkf/Sherrie.Fredrick/C96_replay_winter/2015120218/GFSPRS.GrbF012.grib2

I am not sure of what I need to do in order to get METplus to see my input file.

Many Thanks,
Sherrie

**Comment by georgemccabe** [comment, marked as answer]
Hi @sherrieF, it looks like the value set for FCST_GRID_STAT_INPUT_TEMPLATE has a stray quotation mark (") at the end that is causing the issue. Removing that character should fix the problem. Please let me know if that does not work.

**Reply by jprestop** [reply]
Hi @sherrieF.  One other thing I noticed is that you have set GRID_STAT_CONFIG_FILE to the location of your METplus config file instead of to the location of the MET config file:

```
# Location of MET config file to pass to GridStat

GRID_STAT_CONFIG_FILE = /home/Sherrie.Fredrick/METplus/gridstat/GridStat.conf 
```

I believe you want to have it set to the following instead (unless you have a different MET Grid-Stat config file you'd like to point to:

```GRID_STAT_CONFIG_FILE = {PARM_BASE}/met_config/GridStatConfig_wrapped```

**Reply by sherrieF** [reply]
Thanks Julie and George,
    I am now making some progress.  I am very new to METplus. So your help is very much appreciated.
Sherrie
